src/schema.js

```diff
@@ -9,14 +9,17 @@ import {
   getResourceRef,
 } from './lib/normalize';
 import { request } from './lib/request';
-import { extract } from './utils';
+import { extract, checkIncludesPath } from './utils';
 import { LocationContext } from './location';
 
 const Schema = ({ url, includePath = [] }) => {
   const [type, setType] = useState('');
   const [attributes, setAttributes] = useState([]);
   const [relationships, setRelationships] = useState([]);
-  const { include } = useContext(LocationContext);
+  const { include, toggleInclude } = useContext(LocationContext);
+
+  const includesEnabled = checkIncludesPath(include, includePath);
+  const includePathDisplay = includePath.join('.');
```

Comment:
Maybe this should be `includePathString` because it's not only for display, it's only used for `onChange={() => toggleInclude(includePathDisplay)}`

Reply:
can do, I'll just commit and push this.